tests/unit/helpers/eventsRepetition.test.js

```diff
@@ -444,14 +507,36 @@ describe('createRepetitionsByWeek', () => {
     );
   });
 
-  it('should create repetition every two weeks', async () => {
+  it('should create repetition by week from first event to current time plus 90 days', async () => {
```

Comment:
should create repetition by week with first event in the past

Reply:
done